bucket/winlibs-mingw-ucrt.json

```diff
@@ -1,17 +1,17 @@
 {
     "homepage": "https://winlibs.com/",
-    "description": "winlibs standalone build of GCC compiler and MinGW-w64",
+    "description": "winlibs standalone build of GCC compiler and MinGW-w64 with llvm",
```

Comment:
This manifest does not include LLVM, right?

Reply:
Good catch, missed that description check...

Reply:
Oh yes I forgot to change this line. I will create a new PR fixing this shortly

Reply:
https://github.com/chawyehsu/dorado/pull/446